src/storage/txn/scheduler.rs

```diff
@@ -416,6 +416,7 @@ fn process_read(
     mut cmd: Command,
     ch: SyncSendCh<Msg>,
     snapshot: Box<Snapshot>,
+    sched_ctx: &mut SchedContext,
```

Comment:
why not handle process_write at same time?

Reply:
Because there is nothing need to be local in process_write, only these 3 metrics need to be local.